tox.ini

```
@@ -1,80 +1,83 @@
 [tox]
-envlist =
-        py38-celery{50}-django{32, 40}-drflatest
-        quality,
-        version_check,
-        pii_check,
-        translations
+envlist = 
+    py38-celery{50}-django{32,40,42}-drflatest
+    quality,
+    version_check,
+    pii_check,
+    translations
```

Let's try running translations check with Django 42 as well. 
```suggestion
    translations-django{32, 42}
```

---

done with comment.